docs/topics/http/sessions.txt

```diff
@@ -348,6 +348,16 @@ Bundled serializers
         >>> request.session['0']
         'bar'
 
+
+    By default your serializer in Django does not allow non-UTF8 bytes to be stored in session values, note that using non unicode values for ``request.session`` won't work as expected::
```

Comment:
I don't think the example is particularly needed. I just added it to the ticket so that someone could quickly reproduce if they had other opinions on how to deal with this issue.

I ran into it because my package django-restricted-sessions stores HTTP user agent headers in the session. However, HTTP headers do not have to be valid UTF-8, so when one ended in `\xd9`, that would cause this exception when saving the session. The solution was `force_text` with the right `errors` parameter before I save the UA in the session. So it's quite niche.